app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -412,14 +412,10 @@ const MakeDecision = ({ data }) => {
         shouldShow: isMLSuggestionsAvailable,
         isOpen: activeTab === MACHINE_LEARNING_SUGGESTIONS,
         title:
-          clusterIds.length === 1 ? (
-            <div>
-              {formatMessage(messages.machineLearningSuggestions, {
-                target:
-                  clusterIds.length === 1 ? formatMessage(messages.MLSuggestionsForCluster) : '',
```

Comment:
Do we still need `MLSuggestionsForCluster` and `machineLearningSuggestions` messages?